src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -614,6 +614,14 @@ public function text(/* string $default = null, bool $normalizeWhitespace = fals
 
         $text = $this->getNode(0)->nodeValue;
 
+        if (\func_num_args() <= 1) {
+            if (trim(preg_replace('/(?:\s{2,}+|[^\S ])/', ' ', $text)) !== $text) {
+                @trigger_error(sprintf('"%s()" will normalize whitespaces by default in Symfony 5.0, set the 2nd "$normalizeWhitespace" argument to false to retrieve the non-normalized version of the text.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
`2nd` -> `second` ?

Also, do we mention the argument name explicitly in other error messages? If we don't, we could remove the mention to `$normalizeWhitespace` here. Thanks.

Reply:
After a quick search, I found many examples following the same pattern (2nd and the name of the argument). Example: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/PropertyAccess/PropertyAccessor.php#L855

Reply:
That being said, let me know what's the preferred pattern and I'll update this one :)